### ISSUE TITLE
unable to resolve class groovy.json.JsonSlurper  @ line 1, column 1.

### ISSUE THREAD

**Issue by LmG7119** [nvuillam/vscode-groovy-lint#248]
I have a test Groovy code and it runs but `Lint` is complaining that it can't resolve import. Any idea? 

```
import groovy.json.JsonSlurper

jTest = '[{"key":"value", "key2":"value2"}]'
jsonParser = new JsonSlurper()
jsonObject = jsonParser.parseText(jTest)

println jsonObject.key
println jsonObject.key2
```
